__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const app = require("../app.js");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("App", () => {
+  describe("Invalid Endpoint", () => {
+    it("Returns a 404 status and and and object with a key msg and a value 'Path not found'", () => {
+      return request(app)
+        .get("/wrong-endpoint")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Path not found!");
+        });
+    });
+  });
+
+  describe("GET api/topics", () => {
+    it("Should receive a 200 status and object with a topics key and an array of object topics as a value. Each topic will have the slug and description properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          topics.forEach((topic) => {
```

Comment:
If we are going to use a forEach method when testing, we will also need to check the length of the array to ensure we are still getting the correct amount of elements back. Please add a length check within this test